tickets/package.json

```diff
@@ -18,54 +18,54 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@babel/core": "7.6.0",
-    "@storybook/addon-actions": "5.2.1",
-    "@storybook/addon-knobs": "5.2.1",
-    "@storybook/addon-storyshots": "5.2.1",
-    "@storybook/addon-viewport": "5.2.1",
-    "@storybook/addons": "5.2.1",
-    "@storybook/react": "5.2.1",
-    "@testing-library/jest-dom": "4.2.0",
+    "@babel/core": "7.6.4",
+    "@storybook/addon-actions": "5.2.5",
+    "@storybook/addon-knobs": "5.2.5",
+    "@storybook/addon-storyshots": "5.2.5",
+    "@storybook/addon-viewport": "5.2.5",
+    "@storybook/addons": "5.2.5",
+    "@storybook/react": "5.2.5",
+    "@testing-library/jest-dom": "4.2.2",
     "@testing-library/react": "9.3.1",
-    "@types/jest": "24.0.18",
+    "@types/jest": "24.0.21",
     "@types/next": "8.0.6",
-    "@types/qrcode.react": "0.8.2",
-    "@types/react": "16.9.2",
-    "@types/react-redux": "7.1.2",
+    "@types/qrcode.react": "0.9.0",
+    "@types/react": "16.9.11",
+    "@types/react-redux": "7.1.5",
     "@types/storybook__react": "4.0.2",
-    "@types/styled-components": "4.1.19",
-    "@unlock-protocol/unlock-js": "0.3.20",
```

Comment:
Hum. I think this is a breaking change so I am a bit worried to see this.

Reply:
It's a minor revision, that's what renovate picks up

Reply:
since we mock this and have no integration tests for this we may end up in trouble here. 
Let me at least try to remove that dependency if we do not need it anymore?

Reply:
I can roll it back, let's just let this run through CI once so I can make all the needed changes in one sweep

Reply:
yes, no worries.